src/base.css

```diff
@@ -1357,6 +1359,22 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Styles to support the mention of Unicode characters and character sequences *****/
+   /* For more information, see: https://www.w3.org/TR/international-specs#char_ref  */
+
+	.codepoint bdi {
+		line-height: 1em;
+		font-size: 125%;
+		padding-inline: 0.25em;
+	}
+
+	.uname {
+		font-variant: all-small-caps;
```

Comment:
We restored small caps instead of using `font-size` per fantasai. Of course, this turns up the small caps numbers bug in Chrome (hence the now removed oldstyle-nums--we just turn a blind eye to Chrome's icky appearance for now). Going back to font-size would fix both.